src/controllers/channels.js

```diff
@@ -0,0 +1,15 @@
+var ChannelModel = require('../models/channel');
```

Comment:
It seems that the file model/channel.js does not exit

Reply:
Yeah, it was already present on channel-creation, i didn't add it because the branch needed to rebase on top of it